core/src/main/java/de/danoeh/antennapod/core/feed/FeedMedia.java

```diff
@@ -199,6 +208,18 @@ public void setSize(long size) {
         this.size = size;
     }
 
+    /**
+     * Indicates we asked the service what the size was, but didn't
+     * get a valid answer and we shoudln't check using the network again.
+     */
+    public void setCheckedOnSize() {
```

Comment:
I wound up renaming it to make it a little more clear what's up. I prefer the discrete function so that the functionality is contained within FeedMedia and others don't have to worry about how it's done exactly. (though this still isn't perfect)